src/appkit.cs

```diff
@@ -21879,4 +22792,79 @@ public interface NSAccessibilityLayoutItem : NSAccessibilityGroup {
 		[Export ("setAccessibilityFrame:")]
 		void SetAccessibilityFrame (CGRect frame);
 	}
+
+	[Protocol, BaseType (typeof (NSObject))]
+	public interface NSObjectAccessibilityExtensions {
+		[Obsolete]
```

Comment:
That's not really helpful for developers as it does not say why ? and what's the recommended replacement ?

In most cases that's better handled with something like:

`[Availability (Obsoleted = Platform.Mac_10_10, Message = "Use XXX instead")]`

as it tells when it was obsoleted and how to update the code (when possible). IIRC you mentioned some things were obsolete without an alternative and, if that's the case, then it should become the text, e.g. "No alternative is documented".

Also in some cases it's `Deprecated` and not `Obsoleted` that should be used (or both).